pkg/activitypub/service/spi/spi.go

```diff
@@ -66,6 +66,11 @@ type ProofHandler interface {
 	HandleProof(witness *url.URL, anchorID string, endTime time.Time, proof []byte) error
 }
 
+// LogMonitorHandler registers/un-registers actor's log for monitoring.
+type LogMonitorHandler interface {
```

Comment:
I wonder if this should be called 'AcceptFollowHandler' since the handler decides what action should be performed when a Follow activity is accepted.

Reply:
sure, I can change this

Reply:
@bstasyszyn done